services/topology-engine/queue-engine/topologylistener/context.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2017 Telstra Open Source
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import logging
+import uuid
+
+from topologylistener import const
+from topologylistener import log as log_module
+
+raw_logger = logging.getLogger(__name__)
+
+
+class OperationContext(object):
+    CORRELATION_ID_FIELD = 'correlation_id'
+
+    invalid_correlation_ids = {
```

Comment:
Because that are useless from correlation id point of view. Any "constant" value are invalid value for correlation id. It can be incomplete list of our constant correlation ids. So it should be extended if more such constants be discover.